README.md

```diff
@@ -1,72 +1,77 @@
-# Master Thesis Proposal
-# Comparing and evaluating traditional machine learning algorithms with deep learning algorithms in the field of AMR detection.
-# Author - Varun Sree Bholalayam (00807289)
+# Deep learning vs traditional ML algorithms for AMR detection
+
+Master Thesis Proposal
+
+Author - Varun Sree Bholalayam (00807289)
 
 
 ## Problem Introduction
-***********************
-Antimicrobial Resistance (AMR) occurs when bacteria, viruses, fungi and parasites evolve over time and no longer respond to medicines making infections harder to treat and increasing the risk of disease spread, severe illness and death. AMR is a much overlooked aspect in the field of public health since the predicted mortality rate due to AMR by the year 2050 is more concerning than cancer<sup>1</sup>.
 
-Some examples of so called 'superbugs' are,
+Antimicrobial Resistance (AMR) occurs when bacteria, viruses, fungi and parasites evolve over time and no longer respond to medicines making infections harder to treat and increasing the risk of disease spread, severe illness and death. AMR is a much overlooked aspect in the field of public health since the predicted mortality rate due to AMR by the year 2050 is more concerning than cancer<sup>1</sup>.
 
-1.Drug resistant tuberculosis. 
+Some examples of so called "superbugs" are:
 
-2.Methicillin-resistant Staphylococcus aureus.  
+1. Drug resistant tuberculosis
+2. Methicillin-resistant Staphylococcus aureus
+3. Multidrug-resistant Pseudomonas aeruginosa.
 
-3.Multidrug-resistant Pseudomonas aeruginosa.  
+Both genotypic and phenotypic drug susceptibility tests (DST) are used clinically. Phenotypic DST have limitations including extended turnaround time for slow-growing bacteria such as Mycobacterium tuberculosis (MTB) and bias due to potential contamination<sup>2</sup>.
 
-Both genotypic and phenotypic drug susceptibility tests are used clinically. Phenotypic DST have limitations including extended turnaround time for slow-growing bacteria such as Mycobacterium tuberculosis (MTB) and bias due to potential contamination<sup>2</sup>.
+<!-- I don't know how to connect the previous paragraph -->
```

Comment:
Was trying to introduce the two alternative ways for DST detection. Our dataset is prepared using the Phenotypic DST method. I thought of elaborating the pros and cons of both these approaches. 
Do you think this statement is not significant here?